coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -1614,6 +1614,24 @@ def _add_batch_norm():
         _add_batch_norm()
 
 
+@register_torch_op
+def _weight_norm(context, node):
+    v, g, dim = _get_inputs(context, node, expected=3)
+
+    if dim.val != 0:
```

Comment:
I think we should support different `dim` in the interest of completeness,
the only place to change is the `axes`.
For instance, if the `dim==1` for the rank 3 input,
the axes would be `[0, 2]`, which exlude `1` from `range(3)`

Reply:
Good idea. Done.

Curiously `dim=-1` is the same `dim=None`, which mean calculating the norm over the entire tensor.